tests/default/_core/aggregation/nested.yaml

```diff
@@ -0,0 +1,100 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test aggregation endpoints with nested.
+prologues:
+  - path: movies
+    method: PUT
+    request_body:
+      payload:
+        mappings:
+          properties:
+            a:
+              type: nested
+              properties:
+                b1:
```

Comment:
We plan to use these examples in the documentation. Is there some realistic agg scenario with movies? Maybe actors or something that would differentiate keywords like `nested` from data like `movie`.